app/controllers/web_hooks/slack_controller.rb

```diff
@@ -10,12 +11,19 @@ def build_create_response
     when 'url_verification'
       params[:challenge]
     when 'event_callback'
-      slack_event(params, params[:event][:type])
+      return "Skipping disallowed event #{params[:type][:subtype]}" unless allow_event?(params)
```

Comment:
Looks like a typo here with `params[:type][:subtype]`. May want to just grab `params[:event]` as a variable.